lib/verify.js

```diff
@@ -52,6 +53,12 @@ module.exports = async (pluginConfig, context) => {
   } else if (githubToken && !errors.find(({code}) => code === 'EINVALIDPROXY')) {
     const github = getClient({githubToken, githubUrl, githubApiPathPrefix, proxy});
 
+    // https://github.com/semantic-release/github/issues/182
+    if (githubToken.substr(0, 3) === 'v1.' && env.GITHUB_ACTION) {
```

Comment:
I would rather avoid to depend on the format of the token. So I would only check for `env.GITHUB_ACTION`.

Reply:
I’ve also added a longer comment explaining the early return in case `env.GITHUB_ACTION` is truthy